content/1.2/docs/serverless-workflows/assessment/mta-analysis.md

```diff
@@ -3,4 +3,78 @@ title: MTA Analysis
 date: "2024-02-29"
 ---
 
-{{< remoteMD "https://github.com/parodos-dev/serverless-workflows/blob/main/mta/README.md?raw=true" >}}
+# MTA - migration analysis workflow
+
```

Comment:
we have 1.2.x branch for this repository:
https://github.com/parodos-dev/serverless-workflows/tree/v1.2.x